app/views/application_records/show.haml

```diff
@@ -1,15 +1,16 @@
-%h1 Application Record
+%h1.h2="#{@record.position.name} Application Record"
```

Comment:
And just as a reminder, if you want to string interpolate in HAML, you don't actually need quotes:

```haml
%h1.h2 #{record.position.name} Application Record
```